### PR TITLE
3.4:  Add rocksdb clock fix to known issues

### PR DESCRIPTION
**Pull request by matthewvon**
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-documentation/321/

### CODE REVIEW

**Comment by sleto-it** [comment]
Preview build - I think there are some issues with that <pre> : http://web01.arangodb.biz/bugfix-3.4/mv-rocksdb-knownissue/Manual/ReleaseNotes/KnownIssues34.html

**Comment by matthewvon** [comment]
@sleto-it I have pushed an alternate formatting.  Would you try generating the html again?

**Comment by sleto-it** [comment]
still broken (same link as before) - we could leave this to simran, i believe, should be easy fix for him

**Comment by Simran-B** [comment]
Literal line breaks are simply not supported in Markdown tables and there does not seam to be a way to escape literal line breaks. What does seem to work is an encoded line break `&#13;`.

Note that `> pre` requires a literal line break in Markdown and can thus not be used - you would end up with a literal `> pre` in the output.

So the full solution is: `<pre>multiple&#13;lines</pre> | foo | bar`

**Comment by Simran-B** [comment]
image

File: Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md [hunk 1]
```diff
@@ -36,3 +36,4 @@ Other
...
+| **Date Added:** 2019-02-18 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** There is a clock overflow bug within Facebook's RocksDB storage engine for Windows. The problem manifests under heavy write loads, including long imports. The Windows server will suddenly block all writes for minutes or hours, then begin working again just fine. An immediate workaround is to change the server configuration: <pre><code>[rocksdb]&#13;throttle = false</code></pre> **Affected Versions:** all 3.x versions (Windows only) <br> **Fixed in Versions:** 3.4.4 <br> **Reference:** [facebook/rocksdb#4983](https://github.com/facebook/rocksdb/issues/4983)  |
```
Comment (sleto-it) [inline Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md]: one question is this affecting 3.3 as well? if yes we will need to add the known issue also in the 3.3 page of this 3.4 branch (as unresolved, if the fix is not in 3.3, e.g. "-")

Comment (matthewvon) [reply Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md]: This impacts every version of arangodb that includes rocksdb.  Frank has merged fixes to devel and 3.4 branches.

Comment (sleto-it) [reply Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md]: okay, thanks - then we have to mention to issue in this page as well - https://docs.arangodb.com/3.4/Manual/ReleaseNotes/KnownIssues33.html

Comment (matthewvon) [reply Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md]: Your team doing the PR, or would prefer I do it?

Comment (sleto-it) [reply Documentation/Books/Manual/ReleaseNotes/KnownIssues34.md]: should be an easy push in this PR. No preferences: anyone who is available and can/want

**Comment by matthewvon** [comment]
3.3 text looks good to me.

**Pull request merged**